src/xenia/cpu/hir/hir_builder.cc

```diff
@@ -1872,20 +1872,24 @@ Value* HIRBuilder::AndNot(Value* value1, Value* value2) {
   ASSERT_NON_FLOAT_TYPE(value1);
   ASSERT_NON_FLOAT_TYPE(value2);
   ASSERT_TYPES_EQUAL(value1, value2);
+  //only other type it can be used with is INT64_TYPE (andc)
+  if (value1->type != VEC128_TYPE) {
+    return this->And(this->Not(value2), value1);
+  } else {
+    if (value1 == value2) {
+      return LoadZero(value1->type);
+    } else if (value1->IsConstantZero()) {
```

Comment:
Is there a reason for double ``else if`` instead of using logical or?

Reply:
Nope, changed it